eksconfig/env_test.go

```diff
@@ -570,6 +580,10 @@ func TestEnv(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	// TODO
+	fmt.Println(*cfg.Spec.ClusterAutoscaler)
```

Comment:
Should you be doing an assert statement here instead of a print?

Reply:
Yeah now asserting with `testing.T`